DynamicActiveRecord.php

```diff
@@ -234,16 +234,7 @@ function ($matches) {
      */
     public function __get($name)
     {
-        $value = null;
-        try {
-            $value = parent::__get($name);
```

Comment:
that's a good point and worth explaining. 

    select column_json(column_create('x', null));    >>  {}

so you cannot save a null "value" in a dynamic column. given how SQL null is normally 
defined, "absent", "doesn't exist for this record" and so forth, this seems reasonable to me.

so, for example, if you update a record setting a dynamic column to null, you remove 
the dynamic column from the record.

the idea with DAR is that you can set *anything* in a record and save it. so in 
any given record in the db, *anything* might exist. the app should 
be able to read something and get null back (meaning "not in this record") without 
needing to use `isset()` first or use `try {}`

we changed a basic thing about AR by allowing the user to set undeclared attributes
without getting an exception so is seems ok to me to allow the user to get in a 
similar style.